### PR TITLE
404 page enhanced for DWoC

### PR DESCRIPTION
**Pull request by vedikaag99**
## Issue that this pull request solves

 Closes: #287 (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request
no

@Kajol-Kumari Please check this PR as I was having trouble with the previous one.

### CODE REVIEW

File: frontend/src/components/404/not_found.css [hunk 1]
```diff
@@ -4,12 +4,54 @@
...
+  color: var(--secondary-color);
+}
+
+.back{
+  background: linear-gradient(to right, #121f3a 0%, #2a4886 50%, #162f66 100%);
+  background-size: 200% auto;
+  color: #fff;
+  padding:16px;
+  border-radius:50px;
+  font-size: 21px;
```
Comment (Kajol-Kumari) [inline frontend/src/components/404/not_found.css:34]: let's make it 16px

File: frontend/src/components/404/not_found.css [hunk 2]
```diff
@@ -4,12 +4,54 @@
...
   font-size: 2rem;
+  font-weight: 900;
+  color: var(--secondary-color);
+}
+
+.back{
+  background: linear-gradient(to right, #121f3a 0%, #2a4886 50%, #162f66 100%);
+  background-size: 200% auto;
+  color: #fff;
+  padding:16px;
```
Comment (Kajol-Kumari) [inline frontend/src/components/404/not_found.css:32]: let's make it `padding:8px 16px;`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @vedikaag99 please format the css file and make the page somewhat like this :point_down: 
Screenshot from 2021-01-23 18-34-05

**Comment by Kajol-Kumari** [comment]
Also don't forget the hover effect on the `go Home!` button :)

**Comment by vedikaag99** [comment]
@Kajol-Kumari I have made the required changes. Please check.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Pull request merged**